.travis.yml

```diff
@@ -21,6 +21,15 @@ jobs:
     script:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest
+  - name: Minikube - Frontend
+    cache:
+      directories:
+        - ~/.npm
+        - ~/.cache
+    node_js: 9.4.0
+    script:
+    - scripts/setup-minikube-for-linux.sh
+    - make test CHARTS=enterprise-suite-latest ES_TEST=frontend
```

Comment:
We need to test the stable version (not `-latest`) as well, so we have confidence in our releases.

Reply:
just added :)

Reply:
Hey @jsravn ,  actually we only run the test for latest image is on purpose.  @dbrinegar , @marcoderama and I had a meeting to discuss which image we should run e2e test. We think that there are some feature gap between stable and latest image, so it cannot share the same e2e test because some tests only works for new feature. The right way should be generate/tag the release image from the latest image which pass the test. 

Take https://travis-ci.org/lightbend/helm-charts/builds/431063603 for example,  we found that #333.4 (stable image) is failed. but #333.5 (latest image) is passed.  The reason is that the test is design for latest image.

Reply:
oh~~  I checked it with @dbrinegar ,  he said "if your tests don't work against any released version of a thing, we need a release before it can be merged".  I am going to create a release for es-console.

Reply:
Yup you should depend on a released es-console in helm - the point of which is to tie together the versions. `latest` is just a weak signal of problems, and shouldn't block releases/PRs.

Reply:
Let's make this step only run in the nightly cronjob. Add:

```
if: type in (cron)
```